packages/affiliates/payouts/deployers/2020-11-09_2020-11-16.json

```diff
@@ -0,0 +1,32 @@
+{
+  "config": {
+    "empWhitelist": [
+      "0xaBBee9fC7a882499162323EEB7BF6614193312e3",
+      "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56",
+      "0x306B19502c833C1522Fbc36C9dd7531Eda35862B",
+      "0x3a93E863cb3adc5910E6cea4d51f132E8666654F"
+    ],
+    "startTime": 1604962800000,
+    "endTime": 1605567600000,
+    "totalRewards": "50000",
+    "network": 1
+  },
+  "startBlock": { "timestamp": 1604962804000, "number": 11226178 },
+  "endBlock": { "timestamp": 1605567593000, "number": 11271760 },
```

Comment:
is this correct? should the timestamp not be `1605564000` for `11/16/2020 @ 10:00pm (UTC)`

Reply:
i thought it went to 11pm - 11pm utc. can rerun, but please confirm that. if it goes from 11pm - 10pm we will lose an hour between payouts.

Reply:
Sorry, you are correct on the time range here! It should run from 1ppm to 11pm UTC over the week.

The timestamp you've got runs until 11/16/2020 @ 10:59pm (UTC). So we effectively 11 to 11 returns 11 to 10.99999. This is slightly different to the LM which uses the `block+1` for each subsequent week. In either case, this is interchangeable by 15 seconds on either side so no need to change anything.